### ISSUE TITLE
Using power select in a route

### ISSUE THREAD

**Issue by mmusket** [cibernox/ember-power-select#164]
I'm using ember 2.2 and I'm trying to add a power-select inside of a route. 

At the moment (action "foo") is not supported inside of routes so when i change the onchange method to 
onchange=foo it throws an onchange function is needed. 

Is there a way around this??

**Comment by cibernox** [comment]
Closure action can only be taken from the context of the template (that is a component or a controller). 

You need to define the action in your controller and make that action send the action to the route. This is an ember limitation rather than power-select specific.

This SO thread touches the same topic: http://stackoverflow.com/questions/31101296/how-to-call-an-action-on-a-route-using-closure-actions

The tl;dr; is that it's not possible until routable components are shipped. For the moment you need to create an action in the context and forward the action to the route.
